lib/mix/tasks/phx.gen.html.ex

```diff
@@ -0,0 +1,138 @@
+Code.require_file("../../../installer/lib/phx_new/generator.ex", __DIR__)
```

Comment:
@chrismccord regarding build failures, compiler is handing on these:

```
Compiling lib/mix/tasks/phx.gen.json.ex (it's taking more than 10s)
Compiling lib/mix/tasks/phx.gen.schema.ex (it's taking more than 10s)
Compiling lib/mix/tasks/phx.gen.html.ex (it's taking more than 10s)
```

I was able to reproduce this locally by doing `rm -rf _build && MIX_ENV=test mix compile`. I noticed that if I commented out everything in these 3 mix tasks except for `Code.require_file` line it was still hanging, but if I commented out this one as well it went through. Couldn't find the root cause, but maybe it will point you into the right direction.